internal/controller/ols_console_reconciliator_test.go

```diff
@@ -0,0 +1,146 @@
+package controller
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	consolev1 "github.com/openshift/api/console/v1"
+	openshiftv1 "github.com/openshift/api/operator/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Console UI reconciliator", Ordered, func() {
+
+	Context("Creation logic", Ordered, func() {
+		BeforeAll(func() {
+			console := openshiftv1.Console{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: ConsoleCRName,
+				},
+				Spec: openshiftv1.ConsoleSpec{
+					Plugins: []string{"monitoring-plugin"},
```

Comment:
lightspeed-console-plugin for th entire PR

Reply:
This minics the Console resource in the openshift cluster, it has "monitoring-plugin" activated by default.